lib/src/mt_util.c

```diff
@@ -380,9 +380,12 @@ struct rte_mempool* mt_mempool_create_by_ops(struct mtl_main_impl* impl,
   if (cache_size && (element_size % cache_size)) { /* align to cache size */
     element_size = (element_size / cache_size + 1) * cache_size;
   }
+  char name_with_idx[32];
```

Comment:
Can you give a comment to say 32 is safe enough?

Reply:
done